### DISCUSSION TITLE
Define window dimensions of React Native app

### DISCUSSION THREAD

**Discussion by tmrk** [microsoft/react-native-windows#11050] [Q&A] [answered]
There's probably an easy answer to this but I can't seem to find it in the [docs](https://microsoft.github.io/react-native-windows/docs/getting-started).

Is there a way to **define the dimensions of the app window**?

And perhaps even lock the window size i.e. disable resizing (see another question asking this [here](https://github.com/microsoft/react-native-windows/discussions/11023#discussion-4677448))?

In Proton Native they solved this by introducing a [component](https://proton-native.js.org/#/components/Window) called `Window` - not sure if a similar approach would work for React Native for Windows? Is there already some kind of a wrapper component that would allow this?

**Comment by chrisglein** [comment, marked as answer]
There isn't going to be any direct JS access to this native functionality (since most of React Native core is from mobile applications where this sort of thing isn't relevant), but when you drop into native code you should be able to do these things. I'd search around for "UWP disable window resize" and see what you find. Responses like this for example: https://stackoverflow.com/questions/32212301/is-it-possible-to-restrict-the-user-from-re-sizing-a-uwp-app

**Comment by AstrOOnauta** [comment]
hey @tmrk , did you succeed?
